### DISCUSSION TITLE
/player target selector

### DISCUSSION THREAD

**Discussion by ThatCuteOne** [gnembon/fabric-carpet#1646] [Ideas]
I hope this is the right place for suggestions .
sorry for my bad engish :(

I am coding a Datapack right now , that uses the fake player feature and its not really flexible with target selectors like @ p @ a @ s @ e , obviously it would only work for player actions(use , attack, mount).
At the moment my datapack is really hard coded because of it.

Hope someone could implement my idea.
Have a great Day!

EDIT:Grammar

**Comment by James103** [comment]
See also https://github.com/gnembon/fabric-carpet/issues/768.

**Reply by ThatCuteOne** [reply]
Thanks I didn't saw that
